src/elm/Eth/FunctionArg.elm

```diff
@@ -136,6 +147,19 @@ encodeArg ethType value =
         Bytes ->
             Json.Encode.string value
 
+        Bool ->
+            let
+                lowercaseValue =
+                    String.toLower value
+
+                valueString = 
+                    if lowercaseValue == "false" then
+                        ""
```

Comment:
I'm brand new to Elm, so I'll assume you know what you're doing here, but I would have expected this to return "0" instead of ""

Reply:
it's a funny way the value is being encoded. "".encode comes out to 0x0 while "0".encode comes out 0x1. Odd right?